src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -718,7 +722,14 @@ def __call__(
         if hasattr(self, "final_offload_hook") and self.final_offload_hook is not None:
             self.final_offload_hook.offload()
 
+        workflow = None
+        if return_workflow:
+            signature = inspect.signature(self.__call__)
+            argument_names = [param.name for param in signature.parameters.values()]
+            call_arg_values = inspect.getargvalues(inspect.currentframe()).locals
```

Comment:
nice! does that also get the kwargs values?

Reply:
If so, yes, it does.